js/parking_map_data_control.js

```diff
@@ -10,7 +10,7 @@ L.Control.ParkingInput = L.Control.extend({
 
     const inputPanel = L.DomUtil.create('div', 'panel panel-default', container)
     inputPanel.innerHTML = 
-      '<h3>Wilmington, DE</h3>' +
+      '<h4>Wilmington, DE</h4>' +
```

Comment:
Hey @markabrown87 why the change from `h3` to `h4` here? Is the previous heading on the page `h2` or `h3`? (Asking because it's Accessibility-related)

Reply:
Hi @DavidGinzberg, I don't think there is (or was) an h2 or h3 heading within the DOM structure of the map itself. However, in the OpenDataDe menu, the last header used is h3.

Reply:
Got it. I dug into the topic of web map accessibility a little more and it looks like it's a pretty heavy lift, so we won't worry about that in this PR. I'll finish reviewing this week and we can get this PR live shortly.